model/AI/Properties/informationAboutTraining.md

```diff
@@ -8,8 +8,8 @@ Describes relevant information about different steps of the training process.
 
 ## Description
 
-InformationAboutTraining describes the specific steps involved in the training of the AI model.
-For example, it can be specified whether supervised fine-tuning 
+A free form text that describes the specific steps involved in the training of the AI model.
+For example, it can be specified whether supervised fine-tuning
 or active learning is used as part of training the model.
 
```

Comment:
maybe change to: a detailed explanation of the training process, including any specific techniques, algorithms, or methods used to train the model.  For example,   training data used to train the AI model, including any relevant details about the data source, data quality, and data pre-processing steps, specific training algorithm used to train the AI model, such as stochastic gradient descent, backpropagation, or reinforcement learning, any specific training techniques used to improve the performance or accuracy of the AI model, such as transfer learning, fine-tuning, or active learning and any evaluation metrics used to assess the performance of the AI model during the training process, such as accuracy, precision, recall, or F1 score.

Reply:
Done